cw_core/lib/db/sqlite.dart

```diff
@@ -1,9 +1,14 @@
 
-import 'package:sqflite/sqflite.dart';
+import 'dart:io';
+
+import 'package:sqflite_common_ffi/sqflite_ffi.dart';
 
 late Database db; 
 
 Future<void> initDb({String? pathOverride}) async {
+  if (Platform.isLinux) { 
```

Comment:
this should also include macos and windows.
also, we forgot to test backward compatability on desktop platforms, so please run a quick test on them, by having the production version and then building latest main on top of it

Reply:
Will Test today